release-notes/2.0/2.0.0-commit.md

```diff
@@ -291,16 +291,16 @@ This is a somewhat curated list of commits across the entire .NET Core 2.0 devel
 * [`[a53424c]`](https://github.com/dotnet/cli/commit/a53424c) Fixing an OSx break: "osname: unbound variable"
 * [`[b479bd9]`](https://github.com/dotnet/cli/commit/b479bd9) LOC CHECKIN | cli-master | 20170529
 * [`[bca5317]`](https://github.com/dotnet/cli/commit/bca5317) Update to SetupCrossgen 128 - Coherence 25179
-* [`[b48b7d2]`](https://github.com/dotnet/cli/commit/b48b7d2) .NET Core 2+ Naming and Versioning New file name structure for the runtime and the CLI per: https://github.com/dotnet/designs/issues/2
+* [`[b48b7d2]`](https://github.com/dotnet/cli/commit/b48b7d2) .NET Core 2+ Naming and Versioning New file name structure for the runtime and the CLI per: <https://github.com/dotnet/designs/issues/2>
 * [`[7c8bf26]`](https://github.com/dotnet/cli/commit/7c8bf26) Allow resolution of SDKs without minimum msbuild version file
-* [`[76cf6d4]`](https://github.com/dotnet/cli/commit/76cf6d4) .NET Core 2+ Naming and Versioning New file name structure for the runtime and the CLI per: https://github.com/dotnet/designs/issues/2 1. Branding changes 2. renaming of assets
+* [`[76cf6d4]`](https://github.com/dotnet/cli/commit/76cf6d4) .NET Core 2+ Naming and Versioning New file name structure for the runtime and the CLI per: <https://github.com/dotnet/designs/issues/2> 1. Branding changes 2. renaming of assets
 * [`[8b07dfa]`](https://github.com/dotnet/cli/commit/8b07dfa) Adding the runtime identifier option to dotnet clean.
 * [`[9a669ab]`](https://github.com/dotnet/cli/commit/9a669ab) Updating the Sdk to one that includes the error surfacing work.
 * [`[db68d45]`](https://github.com/dotnet/cli/commit/db68d45) Pinning the MSBuild Minimum version to 15.3.0.
 * [`[1a11756]`](https://github.com/dotnet/cli/commit/1a11756) Addressing code review comments by comparing the versions differently and by changing the name of SDK to .NET Core SDK.
 * [`[75eac0c]`](https://github.com/dotnet/cli/commit/75eac0c) Moving version validation to only when a SDK is not overriden. Also adding a validation for the minimum MSBuild version.
-* [`[61c217c]`](https://github.com/dotnet/cli/commit/61c217c) Fix shared framework deb package name in response to https://github.com/dotnet/core-setup/pull/2461
-* [`[e5c3791]`](https://github.com/dotnet/cli/commit/e5c3791) Update the runtime installer and archive file names in response to https://github.com/dotnet/core-setup/pull/2461
+* [`[61c217c]`](https://github.com/dotnet/cli/commit/61c217c) Fix shared framework deb package name in response to <https://github.com/dotnet/core-setup/pull/2461>
+* [`[e5c3791]`](https://github.com/dotnet/cli/commit/e5c3791) Update the runtime installer and archive file names in response to <https://github.com/dotnet/core-setup/pull/2461>
```

Comment:
```suggestion
* [`[e5c3791]`](https://github.com/dotnet/cli/commit/e5c3791) Update the runtime installer and archive file names in response to https://github.com/dotnet/core-setup/pull/2461
```

Reply:
I think with or without `<>` is good, because visually there's no difference. Might even be easier to ue the short form `#number` in the future?